core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpImportServiceDAO.scala

```diff
@@ -32,25 +43,39 @@ class HttpImportServiceDAO(url: String)(implicit
   val http = Http(system)
   val httpClientUtils = HttpClientUtilsStandard()
 
-  def getImportStatus(importId: UUID,
-                      workspaceName: WorkspaceName,
-                      userInfo: UserInfo
+  // retrieve an import job's status from Import Service
+  override def getImportStatus(importId: UUID,
+                               workspaceName: WorkspaceName,
+                               userInfo: UserInfo
   ): Future[Option[ImportStatus]] = {
-    import ImportServiceJsonSupport._
-    import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 
-    val requestUrl = Uri(url).withPath(Path(s"/${workspaceName.namespace}/${workspaceName.name}/imports/$importId"))
+    val requestUrl =
+      Uri(importServiceUrl).withPath(Path(s"/${workspaceName.namespace}/${workspaceName.name}/imports/$importId"))
 
-    val importStatusResponse: Future[Option[ImportServiceResponse]] = retry[Option[ImportServiceResponse]](when5xx) {
-      () =>
-        executeRequestWithToken[Option[ImportServiceResponse]](userInfo.accessToken)(Get(requestUrl)) recover {
-          case notOK: RawlsExceptionWithErrorReport if notOK.errorReport.statusCode.contains(StatusCodes.NotFound) =>
-            None
-        }
+    doImportStatusRequest(requestUrl, userInfo).map { response =>
+      response.map(statusString => ImportStatuses.withName(statusString.status))
     }
+  }
 
-    importStatusResponse.map { response =>
-      response.map(statusString => ImportStatuses.withName(statusString.status))
+  // retrieve an import job's status from cWDS, and translate its status into the known values for Import Service
+  override def getCwdsStatus(importId: UUID, workspaceId: UUID, userInfo: UserInfo): Future[Option[ImportStatus]] = {
+    val requestUrl =
+      Uri(cwdsUrl).withPath(Path(s"/job/v1/$importId"))
+
+    doImportStatusRequest(requestUrl, userInfo).map { response =>
+      response.map(statusString => ImportStatuses.fromCwdsStatus(statusString.status))
+    }
+
+  }
```

Comment:
I've chosen to NOT use the cWDS client here. This keeps our code paths and models for Import Service and cWDS in sync.